.github/CODEOWNERS

```diff
@@ -319,7 +319,8 @@
 
 # ServiceLabel: %Purview
 # PRLabel: %Purview
-/sdk/purview/                                                        @saxenarahulMs
+/sdk/purview/                                                        @saxenarahulMs @sangamtandon-ms
```

Comment:
looks like sangamtandon-ms isnt a valid user name anymore, should it be removed?